controller/backup_controller.go

```diff
@@ -328,12 +328,12 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 		if _, err := bc.ds.UpdateBackupStatus(backup); err != nil && apierrors.IsConflict(errors.Cause(err)) {
 			log.WithError(err).Debugf("Requeue %v due to conflict", backupName)
 			bc.enqueueBackup(backup)
+			err = nil
```

Comment:
Should we not mask the error message here? It is useful for debugging sometimes. For example, this ticket, you use this error log for the analysis

Reply:
There is a debug message above, good enough?

Reply:
We just add the obj to the queue here if it failed to update.
If we dont mock the err to nil than when returning from reconcile function it will add to the queue again in handleErr function.
I found we mock the err as nil in other controllers as well